code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm

```diff
@@ -145,9 +145,8 @@
 		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/gubbuck(loc)
 	else
 		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/guthen(loc)
-	L.key = key //I don't think this needs logging
-	L.mind = mind
-	L.faction = faction
+	if(mind)
+		mind.transfer_to(L)
```

Comment:
i'm having a braintoot but can this just be replaced with

`mind?.transfer_to(L)`

Reply:
Well, yes it can.